server/src/models/Activity.js

```diff
@@ -1,35 +1,57 @@
-const Model = require("./Model.js");
+const Model = require("./Model.js")
 
 class Activity extends Model {
     static get tableName() {
-        return "activities";
+        return "activities"
     }
 
-    static get jsonSchema(){
+    static get jsonSchema() {
         return {
             type: "object",
             required: ["name"],
             properties: {
                 name: { type: "string", minLength: 2 },
                 description: { type: "string" },
-                location: { type: "string" },
-            },
-        };
+                location: { type: "string" }
+            }
+        }
     }
 
     static get relationMappings() {
-        const { Category } = require("./index.js");
+        const { Category, Review, User } = require("./index.js")
         return {
             category: {
                 relation: Model.BelongsToOneRelation,
                 modelClass: Category,
                 join: {
                     from: "activities.categoryId",
-                    to: "categories.id",
-                },
+                    to: "categories.id"
+                }
+            },
+
+            reviews: {
+                relation: Model.HasManyRelation,
+                modelClass: Review,
+                join: {
+                    from: "activities.id",
+                    to: "reviews.activityId"
+                }
             },
-        };
+
+            users: {
```

Comment:
do we need or use this association? even though the ERD looks like there is a join association, most likely you are not using this and should instead remove it (only define associations/ methods that are being used)

we would use this if we wanted to see the list of all the users who have left reviews for this activity. Most likely we would actually want to know the user who left a given review (not the list of all users who did leave a review)